app/services/reports/acquisition_report_service.rb

```diff
@@ -14,35 +14,54 @@ def initialize(year:, organization:)
     def report
       @report ||= { name: 'Diaper Acquisition',
                     entries: {
-                      'Disposable diapers distributed' => number_with_delimiter(distributed_diapers),
+                      'Disposable diapers distributed' => number_with_delimiter(distributed_disposable_diapers),
+                      'Cloth diapers distributed' => number_with_delimiter(distributed_cloth_diapers),
                       'Average monthly disposable diapers distributed' => number_with_delimiter(monthly_disposable_diapers),
+                      'Average monthly cloth diapers distributed' => number_with_delimiter(monthly_cloth_diapers),
                       'Total product drives' => annual_drives.count,
                       'Disposable diapers collected from drives' => number_with_delimiter(disposable_diapers_from_drives),
+                      'Cloth diapers collected from drives' => number_with_delimiter(cloth_diapers_from_drives),
                       'Money raised from product drives' => number_to_currency(money_from_drives),
                       'Total product drives (virtual)' => virtual_product_drives.count,
                       'Money raised from product drives (virtual)' => number_to_currency(money_from_virtual_drives),
                       'Disposable diapers collected from drives (virtual)' => number_with_delimiter(disposable_diapers_from_virtual_drives),
+                      'Cloth diapers collected from drives (virtual)' => number_with_delimiter(cloth_diapers_from_virtual_drives),
                       '% diapers donated' => "#{percent_donated.round}%",
-                      '% diapers bought' => "#{percent_bought.round}%",
+                      '% diapers purchased' => "#{percent_purchased.round}%",
```

Comment:
These two lines seem to be the only values that combine both disposable and cloth together. Any reason why?

Reply:
The original variable was called total_diapers. So, when I re-worked it I ended up with a cloth diaper variable, a disposable diaper variable, and a total which in my brain was all cloth and all disposable. I was trying to think of what metrics would be most useful, and having the total number of both types of diapers donated, and purchased seemed useful. I can totally change it if I went off the rails!

Reply:
Thanks for thinking ahead!  I did have  a conversation with the NDBN folk about this whole area earlier.   It turns out they don't really care about total diapers -- disposable and cloth are very very different in how they impact people,  because the disposables are a recurring need,  whereas the cloth are a one-time thing.  

So the total diapers values are extraneous,  and should be removed.   

Also, apparently the average number of children receiving cloth diapers is not needed.   We wouldn't use the same # of diapers per child for that, and did not come to a consensus on what the value should be.

I'm not sure about the drives values - I'll need to check with our main business representative about that.

Reply:
(But for now, leave them in)